src/directives/outside-click.js

```diff
@@ -23,9 +26,15 @@ export default {
    * @param {Function} binding.value.handler - The outside click handler function.
    * @param {Array.<String>} binding.value.exclude - The excluded $ref DOM elements.
    * @param {Array.<String>} binding.value.excludeIds - The excluded global DOM id's.
+   * @param {Array.<Element>} binding.value.excludeDOM - The excluded DOM elements.
    * @param {Object} vnode - The vue node object.
    */
   bind(el, binding, vnode) {
+    if (!binding.value) {
+      throw new Error('No event handler defined for v-outside-click.');
+    }
+
+    const handler = typeof binding.value === 'function' ? binding.value : binding.value.handler;
     let userIsScrolling = false;
```

Comment:
todo: we should improve the test for a valid/existing handler.

```suggestion
    const handler = typeof binding.value === 'function' ? binding.value : binding.value?.handler;
    
    if (!handler) {
      throw new Error('No event handler defined for v-outside-click.');
    }

    let userIsScrolling = false;
```